### ISSUE TITLE
Cannot add to gradle

### ISSUE THREAD

**Issue by patrickpissurno** [lapism/search#405]
Hey, thanks for the library.

Could you please provide clearer instructions on how to add to gradle? With the given instructions it fails to find it.

This is my project-level gradle file:
```
// Top-level build file where you can add configuration options common to all sub-projects/modules.

buildscript {
    repositories {
        google()
        jcenter()
        
    }
    dependencies {
        classpath 'com.android.tools.build:gradle:3.4.0'
        
        // NOTE: Do not place your application dependencies here; they belong
        // in the individual module build.gradle files
    }
}

allprojects {
    repositories {
        google()
        jcenter()
    }
}

task clean(type: Delete) {
    delete rootProject.buildDir
}
```

And this is what I've added to my app-level gradle file:

```
dependencies {
    /** here I've ommited some non issue-related stuff **/
    implementation 'com.lapism.search:search: ASAP'
}
```

Looking forward to try and use this library in an upcoming project of mine.

**Comment by lapism** [comment]
Hi. It is not released on Bintray yet because I found some bugs. ASAP is shortcut for As soon as possible :-).

**Comment by chanicheetah** [comment]
When do you think this will be released?

**Comment by lapism** [comment]
Probably until Christmas

**Comment by nksaroj** [comment]
> Probably until Christmas

Eagerly waiting for it :)

**Comment by HaitianZhuu** [comment]
waiting

**Comment by chjaeggi** [comment]
also a wait from here :)

**Comment by lapism** [comment]
This should work :)

dependencies {
    implementation 'com.lapism:search:1.0.0-alpha10'
}

**Comment by lapism** [comment]
I am waiting for approval from jCenter.

dependencies {
implementation 'com.lapism:search:1.0.0'
}

**Comment by lapism** [comment]
Approved and released - 1.0.0
